packages/vscode-extension/src/project/applicationSession.ts

```diff
@@ -311,13 +316,25 @@ export class ApplicationSession implements Disposable {
     }
   };
 
+  private onNetworkEvent = (event: DebugSessionCustomEvent): void => {
+    const method = event.body?.method;
+    if (!method || !isCDPMethod(method)) {
+      Logger.error("Unknown network event method:", method);
+      this.networkBridge.emitEvent("unknownEvent", event.body);
```

Comment:
Is there any reason to emit an event here? Why not just log the error and `return`?